fe/app/components/TeamElementRow/TeamElementRow.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { TeamElementProps } from "../TeamElement/TeamElement.models";
+import TeamElement from "../TeamElement/TeamElement";
+import Divider from "@components/Divider/Divider";
+
+const TeamElementRow = ({
+  teamElements,
+}: {
+  teamElements: TeamElementProps[];
+}) => {
```

Comment:
I know it has only one prop but shouldn't we still have a `models.ts` file to go along with the standard we are using on the rest of the components?